composer.json

```diff
@@ -11,10 +11,10 @@
     }
   ],  
   "require": {
-      "php": ">=7.0.0",
-      "behat/behat": "^3.0",
-      "phpunit/phpunit": ">=6.0 <8.0",
-      "symfony/dependency-injection": "^3.0"
+      "php": ">=7.4.0",

```

Comment:
Could we make this something like `"php": "7 || 8 "` and then we would only need one pull request?